receiver/statsdreceiver/README.md

```diff
@@ -12,27 +12,46 @@ The following settings are required:
 
 - `endpoint` (default = `localhost:8125`): Address and port to listen on.
 
+
+The Following settings are optional:
+
+- `aggregation_interval: 70s`(default value is 60s): The aggregation time that the receiver aggregates the metrics (similar to the flush interval in StatsD server)
+
 Example:
 
 ```yaml
 receivers:
   statsd:
   statsd/2:
     endpoint: "localhost:8127"
+    aggregation_interval: 70s
 ```
 
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
 
 ## Aggregation
 
-Currently the `statsdreceiver` is not providing any aggregation. There are
-ideas such as the [Metrics Transform Processor
-Proposal](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/332)
-that intend to enable control over Metric aggregation in a processor.
-
-An alternative will be to implement some simple aggregation in this receiver.
-
+Aggregation is done in statsD receiver. The default aggregation interval is 60s. The receiver only aggregates the metrics with the same metric name, metric type, label keys and label values. After each aggregation interval, the receiver will send all metrics (after aggregation) in this aggregation interval to the following workflow.
+
+It supports:
+
+Gauge(transferred to double):
+- statsdTestMetric1:500|g|#mykey:myvalue
+statsdTestMetric1:400|g|#mykey:myvalue
+(get the latest value: 400)
+- statsdTestMetric1:500|g|#mykey:myvalue
+statsdTestMetric1:+2|g|#mykey:myvalue
+statsdTestMetric1:-1|g|#mykey:myvalue
+(get the value after calculation: 501)
+
+Counter(transferred to int):
+- statsdTestMetric1:3000|c|#mykey:myvalue
+statsdTestMetric1:4000|c|#mykey:myvalue
+(get the value after incrementation: 7000)
+- statsdTestMetric1:3000|c|#mykey:myvalue
```

Comment:
Please add a TODO somewhere for later when this code is converted to the OTLP data model. I believe the counter events should become Sum data points, with AggregationTemporality=Delta and **Monotonic=False**. We can't be sure statsd counters are monotonic and there are documented cases (across the internet) of systems allowing them to be negative.

Reply:
Places that document statsd negative-counter support:
 https://github.com/influxdata/telegraf/issues/1898
 https://thenewstack.io/collecting-metrics-using-statsd-a-standard-for-real-time-monitoring/
 https://docs.datadoghq.com/developers/metrics/dogstatsd_metrics_submission/#count
 
 No mention of negative counters in the most-official document we have (https://github.com/statsd/statsd/blob/master/docs/metric_types.md#counting), but the way this document describes negative-value gauges specifically implies no concern for negative value counters.